### PR TITLE
Create fr.lua

### PR DESCRIPTION
**Pull request by PitaAuPoulet**
Translation for French people

### CODE REVIEW

**Review by TheFantomas — APPROVED** [review]

File: [esx]/esx_identity/locales/fr.lua [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    ['return_debug_xPlayer_get_sex'] = 'Sexe: %s',
+    ['return_debug_xPlayer_get_dob'] = 'Date de naissance: %s',
+    ['return_debug_xPlayer_get_height'] = 'Taille: %s Inches',
+    ['data_incorrect'] = 'Données invalides, veuillez réessayer.',
+    ['invalid_format'] = 'Format de données non valide, veuillez réessayer.',
+    ['no_identifier'] = '[ESX Identity]\nIl y a eu un problème en chargeant votre personnage !\nCode d’erreur : identificateur manquant\n\nCeci est causé par l’absence de votre identifiant. Veuillez revenir plus tard ou signaler ce problème au propriétaire du serveur.',
+    ['missing_identity'] = '[ESX Identity]\nIl y avait un problème de chargement de votre personnage!\nCode d’erreur : identité manquante\n\nIl semble que votre identité manque, essayez de vous reconnecter.',
+    ['deleted_identity'] = 'Caractère supprimé. Veuillez vous joindre pour créer un nouveau caractère.',
+    ['already_registered'] = 'Vous êtes déjà inscrit.',
+    ['invalid_firstname_format'] = 'Format invalide (First Name): Veuillez réessayer.',
```
Comment (jbdevic) [inline [esx]/esx_identity/locales/fr.lua:33]: First Name --> Prénom

File: [esx]/esx_identity/locales/fr.lua [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    ['invalid_lastname_format'] = 'Format invalide (Last Name): Veuillez réessayer.',
```
Comment (jbdevic) [inline [esx]/esx_identity/locales/fr.lua:34]: Nom de famille

File: [esx]/esx_identity/locales/fr.lua [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    ['invalid_dob_format'] = 'Format invalide (DOB): Veuillez réessayer.',
```
Comment (jbdevic) [inline [esx]/esx_identity/locales/fr.lua:35]: Date de naissance

File: [esx]/esx_identity/locales/fr.lua [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    ['invalid_sex_format'] = 'Format invalide (Sex): Veuillez réessayer.',
```
Comment (jbdevic) [inline [esx]/esx_identity/locales/fr.lua:36]: Sexe

File: [esx]/esx_identity/locales/fr.lua [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+    ['invalid_height_format'] = 'Format invalide (Height): Veuillez réessayer.'
```
Comment (jbdevic) [inline [esx]/esx_identity/locales/fr.lua:37]: Poids

File: [esx]/esx_identity/locales/fr.lua [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+    ['error_debug_xPlayer_get_full_name'] = 'Il y a eu un problème lors de l’obtention de votre nom complet.',
+    ['error_debug_xPlayer_get_sex'] = 'Il y avait un problème en obtenant votre sexe.',
+    ['error_debug_xPlayer_get_dob'] = 'Il y a eu un problème lors de l’obtention de votre date de naissance.',
+    ['error_debug_xPlayer_get_height'] = 'Il y avait un problème lors de l’obtention de votre taille.',
+    ['return_debug_xPlayer_get_first_name'] = 'Prénom: %s',
+    ['return_debug_xPlayer_get_last_name'] = 'Nom de famille: %s',
+    ['return_debug_xPlayer_get_full_name'] = 'Nom complet: %s',
+    ['return_debug_xPlayer_get_sex'] = 'Sexe: %s',
+    ['return_debug_xPlayer_get_dob'] = 'Date de naissance: %s',
+    ['return_debug_xPlayer_get_height'] = 'Taille: %s Inches',
```
Comment (jbdevic) [inline [esx]/esx_identity/locales/fr.lua:26]: Inches --> centimètres

File: [esx]/esx_identity/locales/fr.lua [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+    ['debug_xPlayer_get_full_name'] = 'Renvoie votre nom complet',
+    ['debug_xPlayer_get_sex'] = 'Renvoie votre sexe',
+    ['debug_xPlayer_get_dob'] = 'Renvoie votre date de naissance',
+    ['debug_xPlayer_get_height'] = 'Renvoie votre taille',
+    ['error_debug_xPlayer_get_first_name'] = 'Il y a eu un problème lors de l’obtention de votre prénom.',
+    ['error_debug_xPlayer_get_last_name'] = 'Il y a eu un problème lors de l’obtention de votre nom de famille.',
+    ['error_debug_xPlayer_get_full_name'] = 'Il y a eu un problème lors de l’obtention de votre nom complet.',
+    ['error_debug_xPlayer_get_sex'] = 'Il y avait un problème en obtenant votre sexe.',
+    ['error_debug_xPlayer_get_dob'] = 'Il y a eu un problème lors de l’obtention de votre date de naissance.',
+    ['error_debug_xPlayer_get_height'] = 'Il y avait un problème lors de l’obtention de votre taille.',
```
Comment (jbdevic) [inline [esx]/esx_identity/locales/fr.lua:20]: Il y a eu

File: [esx]/esx_identity/locales/fr.lua [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+    ['debug_xPlayer_get_first_name'] = 'Renvoie votre prénom',
+    ['debug_xPlayer_get_last_name'] = 'Renvoie votre nom de famille',
+    ['debug_xPlayer_get_full_name'] = 'Renvoie votre nom complet',
+    ['debug_xPlayer_get_sex'] = 'Renvoie votre sexe',
+    ['debug_xPlayer_get_dob'] = 'Renvoie votre date de naissance',
+    ['debug_xPlayer_get_height'] = 'Renvoie votre taille',
+    ['error_debug_xPlayer_get_first_name'] = 'Il y a eu un problème lors de l’obtention de votre prénom.',
+    ['error_debug_xPlayer_get_last_name'] = 'Il y a eu un problème lors de l’obtention de votre nom de famille.',
+    ['error_debug_xPlayer_get_full_name'] = 'Il y a eu un problème lors de l’obtention de votre nom complet.',
+    ['error_debug_xPlayer_get_sex'] = 'Il y avait un problème en obtenant votre sexe.',
```
Comment (jbdevic) [inline [esx]/esx_identity/locales/fr.lua:18]: Il y a eu

**Review by jbdevic — CHANGES_REQUESTED** [review]

**Comment by TheFantomas** [comment]
@jbdevic Can you create a pr? The gentleman probably won't be very active, so don't get stuck here :D

**Comment by PitaAuPoulet** [comment]
Sorry, first time i do that, do i have to do something more?

**Comment by jbdevic** [comment]
> Sorry, first time i do that, do i have to do something more?

You need to fix the comment of my review

**Comment by Benzo00** [comment]
No respond after 2 weeks.

**Pull request closed without merging**